src/app/components/SideNav/SideNav.module.css

```diff
@@ -0,0 +1,78 @@
+body {
+  background-color: var(--background-color);
+}
```

Comment:
You like to set global CSS in a CSS module. This effect other components too and should be in your globals.css.